src/client/resources/assets/localization/en-us.json

```diff
@@ -50,8 +50,8 @@
   "minicraft.display.options_display.change_key_bindings": "Change Key Bindings",
   "minicraft.display.options_display.language": "Language",
   "minicraft.display.options_display.resource_packs": "Resource packs",
-  "minicraft.display.popup.enter_confirm": "enter to confirm",
-  "minicraft.display.popup.escape_cancel": "escape to cancel",
+  "minicraft.display.popup.enter_confirm": "Enter to confirm",
+  "minicraft.display.popup.escape_cancel": "Escape to cancel",
```

Comment:
What I'm referring to.

Reply:
I understand now. The change needs to be reflected in only nb-no and ru-ru while other localizations are not done as in files yet.

Reply:
Sorry for taking a while.
So what I need to do is capitalize this keys in every language or reverted? Sorry for not understanding immediately.

Reply:
Only 2 languages have these keys in files for now. This kind of string should always be capitalized as they occupy the whole entry.

Reply:
Yeah, if you cange one language, change them all. You don't need to know the other languages to change the capitalisation.